SocialCareCaseViewerApi/V1/Boundary/Requests/CreateAllocationRequest.cs

```diff
@@ -7,19 +7,28 @@ namespace SocialCareCaseViewerApi.V1.Boundary.Requests
     public class CreateAllocationRequest
     {
         [JsonPropertyName("mosaicId")]
-        public long MosaicId { get; set; }
+        public long? MosaicId { get; set; }
```

Comment:
We have validation rules to check for the presence which will ensure that this value is always there, otherwise we will be getting different type of error from the error stack and that validation rule will never be called.